lib/test/t_LeastSquaresExpansion_std.cxx

```diff
@@ -0,0 +1,158 @@
+//                                               -*- C++ -*-
```

Comment:
The reference C++ unit test file `lib/test/t_LeastSquaresExpansion_std.expout` is missing in the PR.